lib/bridge/MemberListSyncer.js

```diff
@@ -15,6 +15,19 @@ function MemberListSyncer(ircBridge, appServiceBot, server, appServiceUserId, in
     this.appServiceUserId = appServiceUserId;
     this.injectJoinFn = injectJoinFn;
     this._syncableRoomsPromise = null;
+    this._memberLists = {
+        matrix: {
+            //$roomId : {
+            //    id: roomId,
+            //    state: stateEvents,
+            //    realJoinedUsers: [],
+            //    remoteJoinedUsers: []
+            //  }
+        },
+        irc: {
+            //$channel : nick[]
```

Comment:
Thinking about this a bit more: do we stand the chance of racing with ourselves? What happens if the NAMES includes one of the bridge's nicks? We obviously shouldn't be trying to make a matrix entry for that nick. Are we checking for this?